plugins/github-actions/src/components/Widget/Widget.tsx

```diff
@@ -108,3 +108,54 @@ export const Widget = ({
     </InfoCard>
   );
 };
+
+const WidgetListContent = ({
+  error,
+  loading,
+  branch,
+}: {
+  error?: Error;
+  loading?: boolean;
+  lastRun: WorkflowRun;
```

Comment:
aren't you going to pass this into the table or so?

Reply:
removed lastRun parameter from the function in the next commit.